contracts/lido_terra_airdrop_registry/src/contract.rs

```diff
@@ -345,3 +243,45 @@ fn query_airdrop_infos(
         })
     }
 }
+
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> StdResult<Response> {
+    let limit = 10_usize;
```

Comment:
Why 10 exactly? Just in case?

Reply:
there is no particular reason. Actually we have just two records in the airdrops registry and we could migrate these by hand individually. We even have no strong reason to overwrite the records, we could use these as is.

Reply:
Maybe set it to 2 then? To be more clear.

Also it think we need some simple script to check that the contracts work fine right after migration (like we did during Anchor -> Lido migration in `deployment_scripts/deploy_local_with_migration.ts`). Maybe not in this task but anyway